pkg/pipelines/rules/lib/pipeline.rego

```diff
@@ -163,6 +163,13 @@ does_task_match(task, vendor_configs) {
 	count(task.inputs) == 0
 }
 
+does_task_match(task, vendor_configs) {
```

Comment:
In general you should have added same rule but with `not task.inputs` in the end instead of `not task_config.inputs`.
It will work because the pipeline parser initialize the inputs list to be an empty array, so you don't have to add it, but I think we should do that for keep the standard (what if the pipeline-parser package will change the initial value to nil?)

Reply:
I don't agree this is not the standard, this is a bug..